Library/Homebrew/language/python.rb

```diff
@@ -88,6 +88,17 @@ def self.setup_install_args(prefix)
       ]
     end
 
+    def self.rewrite_python_shebang(python_path)
+      Pathname(".").find do |f|
+        regex = %r{^#! ?(/usr/bin/env python([23](.[0-9])?)?|/usr/bin/python)$}
+        maximum_regex_length = "#! /usr/bin/env pythonx.y$".length
```

Comment:
```suggestion
        maximum_regex_length = "#! /usr/bin/env pythonx.yy$".length
```

- handle `python3.10` etc. when they come along